File: conditionals/time.py
# Zoe Jimenez, Getting the time PYTHON
import time

#first instance of time in programming
first_time = time.gmtime()
print(first_time)

# current time in seconds
current = time.time()
print(current) #seconds since Jan 1 1970

#Current date and time like we see it normallly
now = time.ctime(current)
print(now)

#Get just a part of the time
local_time = time.localtime(current)
day = local_time.tm_wday
hour = local_time.tm_hour #Military time (0-23)
print(day)
print(hour)

File: conditionals/time.c
// Zoe Jimenez, getting the time
#include <stdio.h>
#include <time.h>



int main(void){
    // Time since Jan 1 1970
    time_t seconds;


    seconds = time(NULL);
    printf("Seconds since Jan 1 1970 = %d \n", seconds);

// current time
    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    printf("Current time and date is %s\n", asctime(timeinfo));

    //Current hour
    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("%d\n", hour);
    
    return 0;
}